x/arkeo/keeper/manager.go

```diff
@@ -200,6 +278,7 @@ func (mgr Manager) SettleContract(ctx cosmos.Context, contract types.Contract, n
 			if err := mgr.keeper.SendFromModuleToAccount(ctx, types.ContractName, client, cosmos.NewCoins(cosmos.NewCoin(contract.Rate.Denom, remainder))); err != nil {
 				return contract, err
 			}
+			contract.Deposit = contract.Paid
```

Comment:
a little confused by this line, can we add a comment on why this logic is needed.

Reply:
added a comment